deploy/database/data.button.sql

```diff
@@ -1152,3 +1152,315 @@ UPDATE button
+UPDATE button SET flavor_text="Graveyard Greg is a Gaming Geek. His hobbies include collecting games, playing games, and guarding the gaming systems." WHERE name="Graveyard Greg";
+UPDATE button SET flavor_text="Memento-Mori is an evil goth. His hobbies include not playing games, designing unplayable game systems, and trying to defeat the Gaming Guardians." WHERE name="Memento-Mori";
+UPDATE button SET flavor_text="Radical is a deus ex machina. Her hobbies include reading, 80s trivia, and guarding the gaming systems." WHERE name="Radical";
+UPDATE button SET flavor_text="Randarch is the self-proclaimed 'Savior of the Gaming Industry'. His hobbies include washing his hair, bossing people around, and letting other people guard the gaming systems." WHERE name="Randarch";
+UPDATE button SET flavor_text="Scarlet Jester is evil incarnate. His hobbies include destroying gaming systems, manipulating events in his favor, and trying to annihilate the Gaming Guardians." WHERE name="Scarlet Jester";
+UPDATE button SET flavor_text="Shane Hensley is the most evil game designer in existence. His hobbies include making alternate history RPGs, tricking people into thinking he's a nice guy, and trying to defeat the Gaming Guardians." WHERE name="Shane Hensley";
+UPDATE button SET flavor_text="The Flying Squirrel has the proportionate strength, speed, and cheek space of a North American gray squirrel. He likes breaking the law, cavorting with squirrels, hibernating all winter, and beating people up." WHERE name="The Flying Squirrel";
+UPDATE button SET flavor_text="The Japanese Beetle fights for truth, justice, and the American way - as long as it doesn't get in the way of his quest for fame and fortune. He likes tall blondes, sexy communists, and beating people up." WHERE name="The Japanese Beetle";
+UPDATE button SET flavor_text="Charity wants to be a social worker when she grows up. When not making grouchy faces, she likes to beat people up." WHERE name="Charity";
+UPDATE button SET flavor_text="Chastity wants to be an out-of-work actor when she grows up. She likes her ballet lessons and beating people up." WHERE name="Chastity";
+UPDATE button SET flavor_text="Faith wants to be an eschatologist when she grows up. She likes to watch violent cartoons and beat people up." WHERE name="Faith";
+UPDATE button SET flavor_text="Hope wants to be a navy seal when she grows up. Besides eating her broccoli, she likes to beat people up." WHERE name="Hope";
+UPDATE button SET flavor_text="Prudence wants to be an entomologist when she grows up. She likes chloroforming butterflies and beating people up." WHERE name="Prudence";
+UPDATE button SET flavor_text="Temperance wants to be a postal worker when she grows up. She likes small defenseless animals and beating people up." WHERE name="Temperance";
+UPDATE button SET flavor_text="Spells and mystic zaps are what Artax do best. His two best words of power are Abracadabra and beatpeopleup." WHERE name="Artax";
```

Comment:
I don't like this sentence, so i'm not 100% sure either way, but i think it should be "...what Artax *does* best"